diff --git a/node_modules/react-dev-utils/webpackHotDevClient.js b/node_modules/react-dev-utils/webpackHotDevClient.js
index fb515f0..b669fd9 100644
--- a/node_modules/react-dev-utils/webpackHotDevClient.js
+++ b/node_modules/react-dev-utils/webpackHotDevClient.js
@@ -59,11 +59,12 @@ if (module.hot && typeof module.hot.dispose === 'function') {
 // Connect to WebpackDevServer via a socket.
 var connection = new WebSocket(
   url.format({
-    protocol: 'ws',
+    protocol: window.location.protocol === 'https:' ? 'wss' : 'ws',
     hostname: window.location.hostname,
     port: window.location.port,
     // Hardcoded in WebpackDevServer
     pathname: '/sockjs-node',
+    slashes: true,
   })
 );
 
